tests/test_workers/test_tasks/test_build.py

```diff
@@ -595,6 +598,37 @@ def side_effect(_, temp_dir, *args, **kwargs):
         mock_dep_b.assert_not_called()
 
 
+@pytest.mark.parametrize('distribution_scope', ('dev', 'stage', 'prod'))
+@pytest.mark.parametrize('deprecate_bundles', (True, False))
+def test_handle_add_request_raises(
+    distribution_scope, deprecate_bundles,
+):
+    binary_image_config = {'prod': {'v4.5': 'some_image'}}
+    bundles = ['some-bundle:2.3-1', 'some-deprecation-bundle:1.1-1']
+    cnr_token = 'token'
+    organization = 'org'
+    greenwave_config = {'some_key': 'other_value'}
+    deprecation_list = []
+
+    with pytest.raises(IIBError):
+        build.handle_add_request(
```

Comment:
could we map attributes to avoid problems when the next code change happens? It happened to me last time that we had a bug in code because of that